### DISCUSSION TITLE
stickers

### DISCUSSION THREAD

**Discussion by alxndr** [kglw-dot-net/site#4] [Ideas]
everybody loves a good slap

**Comment by alxndr** [comment]
stick a QR code on it somewhere? 
![QR code for the domain name](https://user-images.githubusercontent.com/174307/200132289-e8f6f4d6-a6e3-4828-a673-918cc4ee3949.png)

**Reply by Hina-Iluvataro** [reply]
"Peer into 
[QR CODE HERE]
the Gizzverse"
*Wrapped inside a Nonagon*

Also just the reg kglw.net logo slap, with diff color variants. Maybe one made up to look like an old browser site (ie; aol)
